src/services/external/transaction.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Resource } from "@companieshouse/api-sdk-node";
+import ApiClient from "@companieshouse/api-sdk-node/dist/client";
+import { logger } from "../../utils/logger";
+import { Transaction } from "@companieshouse/api-sdk-node/dist/services/transaction/types";
+
+
+export class TransactionService {
+    constructor(private apiClient: ApiClient) {}
+
+    async postTransactionRecord(companyNumber: string, reference: string, description: string): Promise<Transaction> {
+        const transactionRecord = this.createTransactionRecord(companyNumber, reference, description);
+
+        logger.debug(`Created Transaction Record using Company Number: ${companyNumber}, Reference: ${reference}, Description: ${description}`);
+
+        const transactionService = this.apiClient.transaction;
+        const transactionServiceResponse = await transactionService.postTransaction(transactionRecord);
```

Comment:
Could this be one line
```suggestion
        const transactionServiceResponse = await this.apiClient.transaction.postTransaction(transactionRecord);
```